start.sh

```diff
@@ -272,15 +272,15 @@ while [[ $# -gt 0 ]]; do
         echo -e "  -d,   --debug\t\t\t Run in debug mode"
         echo -e "  -h,   --help\t\t\t Show this message"
         echo -e ""
-        echo -e "<----------   BUX SERVER SECTION"
-        echo -e "  -bs,  --bux-server\t\t Whether the bux-server should be run - true/false"
+        echo -e "<----------   SPV WALLET SECTION"
+        echo -e "  -sw,  --spv-wallet\t\t Whether the spv-wallet should be run - true/false"
         echo -e "  -db,  --database\t\t Define database - postgresql, mongodb, sqlite"
         echo -e "  -c,   --cache\t\t\t Define cache storage - freecache(in-memory), redis"
         echo -e "  --xpub\t\t\t Define admin xPub"
         echo ""
-        echo -e "<----------   BUX WALLET SECTION"
-        echo -e "  -bwf,  --bux-wallet-frontend\t Whether the bux-wallet-frontend should be run - true/false"
-        echo -e "  -bwb,  --bux-wallet-backend\t Whether the bux-wallet-backend should be run - true/false"
+        echo -e "<----------   SPV WALLET COMPONENT SECTION"
+        echo -e "  -wf,  --wallet-frontend\t Whether the wallet-frontend should be run - true/false"
+        echo -e "  -wb,  --wallet-backend\t Whether the wallet-backend should be run - true/false"
         echo -e "  --xprv\t\t\t Define admin xPriv"
         exit 1;
```

Comment:
we should also add a section for Block Headers Service, because we forgot to mention -bhs flag, and maybe in the future we will introduce possibility to run it on postgres